src/website/src/pages/ec/components/list/docs/code.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Playground, Html } from '@ecl/website-components';
+import {
+  listOrdered,
+  listUnordered,
+  listUnorderedNoBullet,
+  listUnorderedWithDivider,
+  listDescription,
+  listDescriptionHorizontal,
+} from '../demo';
+
+## Ordered list
+
+<Playground system="ec" selectedKind="components-list" selectedStory="ordered">
+  <Html markup={listOrdered} />
+</Playground>
+
+## Unordered list
+
+<Playground
+  system="ec"
+  selectedKind="components-list"
+  selectedStory="unordered"
```

Comment:
Please double-check settings in playground components. 
Clicking on the first one, it leads to `/?path=/story/components-list--ordered`, whereas the actual story is at `/?path=/story/components-list-ordered-list--default`